### ISSUE TITLE
Paraseptal emphysema

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6702]
**Preferred term label:**
Paraseptal emphysema

**Synonyms**
Distal acinar emphysema

**Definition (free text, please give PubMed ID)**
Paraseptal emphysema is characterized by predominant involvement of the distal alveoli and their ducts and sacs. It is characteristically bounded by any pleural surface and the interlobular septa. This emphysema is characterized by subpleural and peribronchovascular regions of low attenuation separated by intact interlobular septa, sometimes associated with bullae.

PMID:18195376

Comment
See Figure 47 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983  vs Emphysema

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
